### ISSUE TITLE
Configuration File Conflict - Pages and or Missing Name / Type

### ISSUE THREAD

**Issue by Inysal** [dmulloy2/SwornRPG#34]
The Configuration Files that you have on:

https://dev.bukkit.org/bukkit-plugins/swornrpg/pages/sworn-rpg-configuration/
and
https://github.com/dmulloy2/SwornRPG/blob/master/src/main/resources/config.yml

The Bukkit Site is out of date, however; there are a few configuration settings that I don't see on the github site. Were these removed, or do they need to be manually added in?

updatechecker
cooldown radius - If a player is within x radius of a mob spawner, cancel xp gain
money amountperlevel - 

Also this URL is Broken: 
https://github.com/dmulloy2/SwornRPG/blob/master/config.yml

This URL was found on the top of the page of: 
https://dev.bukkit.org/bukkit-plugins/swornrpg/pages/sworn-rpg-configuration/

**Comment by dmulloy2** [comment]
Yeah I don't really update BukkitDev anymore, I should probably get around to it. The config should be pretty well documented as it is though.

**Comment by Inysal** [comment]
That's understandable. I know alot of Spigot Users also still check Bukkit, because Bukkit Plugins work on Spigot. lol As well it was my number one result when I searched google, looking for Spigot Plugins. xD
